### PR TITLE
Feature create routes 160670462

### PR DESCRIPTION
**Pull request by GransonO**
## What does this PR do?
This PR adds the file structures for the API

## Description of Task to be completed?
Create all folder structures and __init__.py files for specific folders(to creates packages). Add a run.py file from which the app shall be started, add a config file for the different server states. Finally add a routes for the different end points. 

## How should this be manually tested?
The project can be forked from the repo. Install all packages from the requirements.txt file and from your command line, run the run.py file to start the server. Using post man, test the endpoint

## What are the relevant pivotal tracker stories?
Pivotal tracker story ID :  [2198167](https://www.pivotaltracker.com/n/projects/2198167)

### CODE REVIEW

File: server/app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+from instance.config import App_Configurations
+
+def create_app(Config_Status):
+    app = Flask(__name__)
+    app.config.from_object(App_Configurations[Config_Status])
+    app.config.from_pyfile('../instance/config.py')
+
+    return app
```
Comment (mwaz) [inline server/app/__init__.py]: add a blank line at the end of this file

File: server/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.pytest_cache/
+*.pyc
+ff-env/
+__pycache__/
```
Comment (mwaz) [inline server/.gitignore]: Add a blank line at the end of this file

File: server/app/__init__.py (hunk 1 above)
Comment (GransonO) [reply server/app/__init__.py]: Done as directed

File: server/.gitignore (hunk 2 above)
Comment (GransonO) [reply server/.gitignore]: Done

**Comment by GransonO** [comment]
I have not added the tests to develop thus the travis-ci builds are bound to fail

**Comment by GransonO** [comment]
Closed as new PR created

**Pull request closed without merging**